### PR TITLE
Update test boilerplate for running on travis

### PR DESCRIPTION
**Pull request by bgrins**
This should be running an in-code and external test in CI once landed

### CODE REVIEW

**Comment by bgrins** [comment]
Looks like it worked: https://travis-ci.org/mozilla/datomish/builds/184625808

File: Cargo.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[root]
```
Comment (rnewman) [inline Cargo.lock]: We shouldn't check in this file.

http://doc.crates.io/guide.html#cargotoml-vs-cargolock

> If you’re building a library that other projects will depend on, put Cargo.lock in your .gitignore. If you’re building an executable like a command-line tool or an application, check Cargo.lock into git. If you're curious about why that is, see "Why do binaries have Cargo.lock in version control, but not libraries?" in the FAQ.

Comment (bgrins) [inline Cargo.lock]: Good catch, i'll remove it

**Comment by bgrins** [comment]
This uses the `tests` directory to run tests out of band: ahttps://doc.rust-lang.org/book/testing.html#the-tests-directory.  Apparently subfolders don't get run automatically so we could move the js tests into this folder if we don't want to have both `test` and `tests` top level dirs

**Comment by bgrins** [comment]
Pushed up the changes

**Pull request merged**